src/index.ts

```diff
@@ -5,6 +5,11 @@ import open from 'open';
 import sendNotification from './notification';
 import {Logger} from './logger';
 
+/**
+ * Send test email
+ */
+sendNotification('test');
+
```

Comment:
I think we should wrap this behind some sort of environment variable or argument. Just so we don't spam.

Take a look at `config.ts` and see if you want to make an environment variable!

Reply:
added an emailTest flag to .env and the config file. Added a test if flag is true to push the notification out. Flag defaults to 0.